### ISSUE TITLE
module not working on 0.8.7

### ISSUE THREAD

**Issue by dirusulixes** [vtt-lair/player-pronouns#5]
fvtt 0.8.7
module 0.1.2
no other modules enabled.

The option to choose pronouns is not shown in the player menu when it is enabled in the module. 
There seems to be an error thrown when the renderPlayList is hooked:

```
VM22912:8 TypeError: Cannot read property 'id' of undefined
    at renderPlayerList (player-pronouns.js:174)
    at Function._call (eval at <anonymous> (libWrapper-error_listeners.js:56), <anonymous>:4:14)
    at Function.call (foundry.js:179)
    at PlayerList._render (foundry.js:2097)
```

**Comment by vtt-lair** [comment]
Thanks, will quickly see if I can get it fixed

**Comment by vtt-lair** [comment]
@dirusulixes Fix has been released as 0.1.3. Please let me know if you are still getting that error on the console. I wasn't able to replicate it, but I added a defensive check to it.

The player config menu now shows the pronouns again.
